packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md

```diff
@@ -22,14 +22,14 @@ Validators are eligible for epoch payments if they proposed at least one block d
 
 - The annual cost of running and securing a validator node, including overhead: 60,000 Celo Dollar
 - Profit margin: $$20\%$$
-- Flat fee to validators : $$annual\_val\_fee = \frac{60,000}{1 - 0.2} = 75,\!000$$ Celo Dollar
+- Flat fee to validators : $$annual_val_fee = \frac{60,000}{1 - 0.2} = 75,\!000$$ Celo Dollar
```

Comment:
Are these underscores going to appear correctly? We escaped them since by default they make subscripts in LaTeX.

Reply:
Ditto multiple spots below.

Reply:
must be my editor :-( reverted

Reply:
Looks like your editor is very stubborn - doesn't appear to be reverted?